atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/CellsSweeper.java

```diff
@@ -98,4 +109,10 @@ public void sweepCells(
             persistentLockManager.releasePersistentLock();
         }
     }
+
+    private String getLoggingMessageFromMap(Multimap<Cell, Long> cellTsPairsToSweep) {
```

Comment:
I would either sort the entries by their key or use the `Multimap`'s own `toString` method.  Fr large maps this output may be hard to read if there are many values associated with the key